### PR TITLE
E2E Test for Partial Refunds

### PR DESCRIPTION
**Pull request by rodelgc**
Closes https://github.com/Automattic/woocommerce-payments/issues/2212

#### Changes proposed in this Pull Request

Title: E2E Test for the `Merchant / Order / Partial refunds (one, several)` critical flow

Description: This PR contains two test cases for partial refunds:
1. A test case where one of two line items is partially refunded by refunding a portion of its cost, like shown below:
    Screen Shot 2021-07-22 at 11 41 57 PM
2. A test case where two of three line items are refunded. The first line item is fully refunded, while in the second line item, only one piece was refunded. Something like below:
    Screen Shot 2021-07-22 at 11 43 45 PM

#### Testing instructions
1. Setup the e2e testing environment as usual.
2. Run the newly added test: `npm run test:e2e ./tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js`. These two tests should pass:
    Screen Shot 2021-07-23 at 12 10 23 AM

### CODE REVIEW

File: tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+/**
+ * Internal dependencies
+ */
+import { merchantWCP } from '../../utils';
+import { fillCardDetails, setupProductCheckout } from '../../utils/payments';
+
+const card = config.get( 'cards.basic' );
+const product1 = config.get( 'products.simple.name' );
+const product2 = 'Belt';
```
Comment (naman03malhotra) [inline tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js:16]: For my learning: where we have stored this data config?  Is it `sample_products_override.csv`?

**Review by naman03malhotra — APPROVED** [review]
LGTM! :shipit: Nice...left a question for my learning.

File: tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js (hunk 1 above)
Comment (rodelgc) [reply tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js:16]: Thanks for reviewing, @naman03malhotra!

> where we have stored this data config

Every time we call `config.get('...')`, we're getting the object from the [test.json](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e/config/test.json) file.

As for the 'Belt' and the 'Hoodie', I just arbitrarily selected them from the pre-existing products in the store.

**Comment by rodelgc** [comment]
Hmm, the tests failed after updating it with`develop`. I'll investigate and fix it tomorrow.

**Pull request merged**